contracts/core/exchange-wrappers/ZeroExExchangeWrapper.sol

```diff
@@ -71,117 +71,118 @@ contract ZeroExExchangeWrapper is
 
     /**
      * IExchangeWrapper interface delegate method.
-     * Parses 0x exchange orders and transfers tokens from taker's wallet.
+     * Parses 0x exchange orders and executes them for Set component tokens
      *
      * TODO: We are currently assuming no taker fee. Add in taker fee going forward
      *
-     * @param  _maker          Unused address of issuance order signer to conform to IExchangeWrapper
-     * @param  _taker          Unused address of fillOrder caller to conform to IExchangeWrapper
-     * @param  _orderCount     Amount of orders in exchange request
-     * @param  _ordersData     Byte string containing (multiple) 0x wrapper orders
-     * @return address[]       Array of token addresses executed in orders
-     * @return uint256[]       Array of token amounts executed in orders
+     * -- Unused address of issuance order signer to conform to IExchangeWrapper --
+     * -- Unused address of fillOrder caller to conform to IExchangeWrapper --
+     * @param  _makerToken          Address of maker token used in exchange orders
+     * @param  _makerAssetAmount    Amount of issuance order maker token to use on this exchange
+     * @param  _orderCount          Amount of orders in exchange request
+     * @param  _ordersData          Byte string containing (multiple) 0x wrapper orders
+     * @return address[]            Array of Set component token addresses executed in orders
+     * @return uint256[]            Array of Set component token amounts executed in orders
      */
     function exchange(
-        address _maker,
-        address _taker,
+        address,
+        address,
+        address _makerToken,
+        uint256 _makerAssetAmount,
         uint256 _orderCount,
         bytes _ordersData
     )
         external
         onlyAuthorized
         returns (address[], uint256[])
     {
-        address[] memory takerTokens = new address[](_orderCount);
-        uint256[] memory takerAmounts = new uint256[](_orderCount);
+        // Ensure the taker token is allowed to be transferred by ZeroEx Proxy
```

Comment:
I know what you mean, but this is a slightly confusing comment given that the `_makerToken` is passed in.

Reply:
+1